server/src/group/group.gateway.ts

```diff
@@ -24,6 +25,7 @@ export class GroupGateway {
   constructor(
     private clientService: ClientService,
     private groupService: GroupService,
+    private eventService: EventService,
```

Comment:
You added EventService here, but GroupModule does not import EventModule in its 'imports' array. This code will not start if you were to run 'docker compose up'.